### PR TITLE
Reworked and fixed versions list

### PR DESCRIPTION
**Pull request by felixoi**
Current status:
preview
preview
preview

### CODE REVIEW

**Comment by phase** [comment]
Relies on #337

**Comment by phase** [comment]
Might want to squash these commits with some better messages.

**Comment by felixoi** [comment]
@phase I'm always squashing my commits before merge ;)

File: public/javascripts/versionList.js [hunk 1]
```diff
@@ -36,54 +37,138 @@ var page = 1;
...
+            // Prev & Next Buttons
+            pagination.find('.next').click(function () {
+                if (TOTAL_VERSIONS / VERSIONS_PER_PAGE > page) {
+                    loadVersions(1);
+                }
+            });
+
+            pagination.find('.prev').click(function () {
+                if (page > 1) {
+                    loadVersions(-1)
```
Comment (lol768) [inline public/javascripts/versionList.js]: `;`

**Comment by felixoi** [comment]
Should be ready now.

**Pull request merged**